src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDao.java

```diff
@@ -192,43 +189,20 @@ public void deleteDirectoryEntriesFromCollection(Firestore firestore, String col
         DocumentSnapshot apply(Transaction xn) throws InterruptedException;
     }
 
-    // Returns null if not found - upper layers do any throwing
-    private FireStoreDirectoryEntry runTransactionWithRetry(Firestore firestore,
-                                                            LookupFunction lookupFunction,
-                                                            String transactionOp,
-                                                            String warnMessage) throws InterruptedException {
-        for (int i = 0; i < LOOKUP_RETRIES; i++) {
-            try {
-                ApiFuture<FireStoreDirectoryEntry> transaction =
-                    firestore.runTransaction(
-                        xn -> {
-                            DocumentSnapshot docSnap = lookupFunction.apply(xn);
-                            if (docSnap == null) {
-                                return null;
-                            }
-                            return docSnap.toObject(FireStoreDirectoryEntry.class);
-                        });
-
-                return fireStoreUtils.transactionGet(transactionOp, transaction);
-            } catch (Exception ex) {
-                if (FireStoreUtils.shouldRetry(ex)) {
-                    logger.warn("Retry-able error in firestore future get - " +
-                        warnMessage + " message: " + ex.getMessage());
-                } else {
-                    throw new FileSystemExecutionException(transactionOp + " execution exception", ex);
-                }
-            }
-
-            TimeUnit.SECONDS.sleep(1);
-        }
-        throw new FileSystemExecutionException(transactionOp + " failed - no more retries");
-    }
+    private static final int SLEEP_BASE_MILLISECONDS = 1000;
```

Comment:
It's a minor thing but I would use a duration type that lets me express the value in fewer digits. In this case it seems like seconds would be a more natural duration type to use.
```suggestion
    private static final int SLEEP_BASE_SECONDS = 1;
```

Reply:
I think we've discussed this before! https://github.com/DataBiosphere/jade-data-repo/pull/853/files#r584991592 
I'll go ahead and switch to seconds. Hopefully that will prevent another bug like you caught in one of your other comments.